### PR TITLE
Update WASM build instructions

### PR DESCRIPTION
**Pull request by SET001**

### CODE REVIEW

**Comment by jdm** [comment]
This change would have helped me! Could we merge it please?

File: src/platforms/wasm.md [hunk 1]
```diff
@@ -80,13 +80,21 @@ It will automatically run a minimal local webserver and open your game in your b
 
 `wasm-bindgen` is the tool to generate all the files needed to put the game on your website.
 
+Install:
+
+```sh
+cargo install wasm-bindgen-cli
+```
+
```
Comment (toolness) [inline src/platforms/wasm.md:88]: Thanks for adding this, I got stuck without it!

**Comment by toolness** [comment]
Hi @inodentry, this PR would have helped me get wasm up and running, as I got stuck with the existing instructions.  Just echoing @jdm's suggestion to merge this!

(Also hi @jdm!)

**Comment by inodentry** [comment]
This stuff is documented properly on a separate page now.

**Pull request closed without merging**